shiny/module.py

```diff
@@ -24,25 +24,85 @@
 _: Id  # type: ignore
 
 
-@no_example()
+@add_example(ex_dir="api-examples/Module")
 def ui(fn: Callable[P, R]) -> Callable[Concatenate[str, P], R]:
+    """Decorator for defining a Shiny module UI function.
```

Comment:
And same for the other docstring. Not sure if this will fix the formatting issues you're seeing, but it's a convenient we follow elsewhere

```suggestion
    """
    Decorator for defining a Shiny module UI function.
```